qualtran/conftest.py

```diff
@@ -75,10 +75,30 @@ def assert_equivalent_bloq_example_counts_for_pytest(bloq_ex: BloqExample):
         raise bce from bce
 
 
+def assert_bloq_example_serialize_for_pytest(bloq_ex: BloqExample):
+    if bloq_ex.name in [
+        'prep_sparse',
+        'thc_prep',
+        'modmul_symb',
+        'modexp',
+        'modexp_small',
+        'modexp_symb',
+        'apply_z_to_odd',
+        'lp_resource_state_symbolic',
+    ]:
+        return
```

Comment:
`pytest.xfail` might be more appropriate here

Reply:
done